pkg/master/master.go

```diff
@@ -124,6 +125,9 @@ type Config struct {
 	// Used to start and monitor tunneling
 	Tunneler genericapiserver.Tunneler
 
+	// Enables using master leases
+	EnableLeases bool
```

Comment:
I'd prefer it to be in its own packages, for sure.  The concern here was to
keep the delta on 1.3 as minimal as possible for easy review.

On Tue, May 31, 2016 at 3:54 PM, Nikhil Jindal <notifications@github.com>
wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/26574#discussion_r65250999>
> :
>
> > @@ -124,6 +125,9 @@ type Config struct {
> >  	// Used to start and monitor tunneling
> >  	Tunneler genericapiserver.Tunneler
> >
> > +	// Enables using master leases
> > +	EnableLeases bool
>
> Why not put it in genericapiserver?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26574/files/2eb2c0c6f09ee226ce04c18219b5b26992bbdb63#r65250999>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p257_RMxDs4mBV25qQjIBkC6Q191ks5qHJIKgaJpZM4IqwBo>
> .
>